protocol/network-types/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
+name = "mina-network-types"
+version = "0.1.0"
+
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+versioned = {path = "../versioned"}
+serde = {version = "1", features = ["derive"]}
+mina-crypto = {path = "../../crypto"} # TODO: Try and remove this dep
+
+[dev-dependencies]
+wasm-bindgen-test = "0.3"
```

Comment:
Where is `wasm-bindgen-test` used? Plz note that wasm tests in new crates need extra setup in CI workflow yaml